BFWControls/Modules/NibView/View/NibTextField.swift

```diff
@@ -112,10 +112,8 @@ open class NibTextField: UITextField {
 private extension UIView {
     
     var subviewTextField: UITextField? {
-        return subviews.first
-            { subview in
-                subview is UITextField || subview.subviewTextField != nil
-            } as? UITextField
+        let textFields = subviews.flatMap { $0 as? UITextField ?? $0.subviewTextField }
+        return textFields.first
     }
```

Comment:
Can we use `subviews.first {...}` rather than `subviews.flatMap {...}.first` since `first {...}` will exit as soon as it finds a match, rather than traversing the entire tree unnecessarily.